pkg/ec2metadata/ec2metadata.go

```diff
@@ -55,7 +55,9 @@ func CheckForSpotInterruptionNotice(metadataURL string, nodeTerminationGracePeri
 		return false
 	}
 	var instanceAction InstanceAction
+
 	json.NewDecoder(resp.Body).Decode(&instanceAction)
+	log.Printf("Spot Instance Interruption Warning received. Instance metadata: %v", instanceAction)
```

Comment:
this needs a newline character added. 

Also, since we're printing the struct, we probably want `%+v` to print field names